pragma-node/src/infra/repositories/onchain_repository.rs

```diff
@@ -316,6 +319,7 @@ pub async fn routing(
     timestamp: TimestampParam,
     aggregation_mode: AggregationMode,
     is_routing: bool,
+    chunk_interval: ChunkInterval,
```

Comment:
we should probably create a struct here instead of all these params

Reply:
done